src/extensions/scratch3_music/index.js

```diff
@@ -102,6 +91,17 @@ class Scratch3MusicBlocks {
         });
     }
 
+    _fetchSounds () {
+        if (!this._assetData) {
+            this._assetData = new Promise((resolve, reject) => {
+                require.ensure([], () => {
```

Comment:
- Something without require.ensure: should throw an error trying to call an undefined function or derefencing a member off of an undefined variable. In those cases the wrapping promise constructor will reject the promise and the caller of `_fetchSounds` can handle the error with a catch handler on the promise.
- Webpack supports require.ensure for node. If you require the node target built script, this will load the sounds successfully.